plugins/woocommerce/client/blocks/assets/js/data/checkout/actions.ts

```diff
@@ -71,21 +71,49 @@ export const __internalSetHasError = ( hasError = true ) => ( {
 } );
 
 /**
- * Used when any of the totals, taxes, shipping, etc need to be calculated, the `calculatingCount` will be increased
- * A `calculatingCount` of 0 means nothing is being updated.
+ * Signals the start of a singular calculation process for totals, taxes,
+ * shipping, etc. Increases the `calculatingCount` which tracks ongoing
+ * calculations. A `calculatingCount` of 0 means nothing is being updated.
  */
-export const __internalIncrementCalculating = () => ( {
+export const __internalStartCalculation = () => ( {
 	type: types.INCREMENT_CALCULATING,
 } );
 
 /**
- * When any of the totals, taxes, shipping, etc are done beign calculated, the `calculatingCount` will be decreased
- * A `calculatingCount` of 0 means nothing is being updated.
+ * Signals the completion of a singular calculation process for totals, taxes,
+ * shipping, etc. Increases the `calculatingCount` which tracks ongoing
+ * calculations. A `calculatingCount` of 0 means nothing is being updated.
  */
-export const __internalDecrementCalculating = () => ( {
+export const __internalFinishCalculation = () => ( {
 	type: types.DECREMENT_CALCULATING,
 } );
 
+/**
+ * @deprecated Use disableCheckoutFor thunk instead
```

Comment:
I think its common practice to also include the WC version we deprecated this function too:

`@deprecated since 9.9.0: Use disableCheckoutFor thunk instead` for example.

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/57566/commits/a211f2995a9328bfecfa61339536cc925e3be2df, I also did the same to `__internalSetExtensionData` after verifying that it targets `9.9.0` too.